### PR TITLE
Update xml doc for SecurityHelper

### PR DESCRIPTION
**Pull request by JunTaoLuo**
This is blocking testing IISIntegration

### CODE REVIEW

File: src/Microsoft.Extensions.SecurityHelper.Sources/SecurityHelper.cs [hunk 1]
```diff
@@ -17,7 +17,8 @@ internal static class SecurityHelper
         /// Merges a new claims principal, placing all new identities first, and eliminating
         /// any empty unauthenticated identities from context.User
         /// </summary>
-        /// <param name="identity"></param>
+        /// <param name="existingPrincipal">The ClaimsPrincipal containing existing ClaimsIdentities.</param>
```
Comment (Eilon) [inline src/Microsoft.Extensions.SecurityHelper.Sources/SecurityHelper.cs]: Can the typenames be changed to be type `<see ...>` elements or whatever that thing is?

**Comment by Tratcher** [comment]
:shipit:

**Comment by JunTaoLuo** [comment]
Seems like @pranavkm fixed this in https://github.com/aspnet/Common/commit/ea62fc778fac35f8de7d7aac4da0d56e2d5790f8

**Comment by pranavkm** [comment]
I added empty nodes. Feel free to merge this in since it has useful comments.

**Pull request merged**